src/css-grids/cssgrids.css

```diff
@@ -0,0 +1,185 @@
+/* Reset & Utilities */
+*{
+   padding: 0;
+   margin: 0;
+   box-sizing: border-box;
+}
+body{
+   font-family: Arial, Helvetica, sans-serif;
+   color: #ccc;
+   background: rgb(20, 0, 0);
+   width: 100vw;
+   height: 100vh;
+   position: relative;
+   text-shadow: 3px 3px rgb(46, 38, 2);
+}
+body::before{
+   content: "";
+   background: url('./img/bg.jpg') no-repeat center center/cover;
+   opacity: 0.4;
+   position: absolute;
+   top: 0;
+   left: 0;
+   width: 100%;
+   height: 100%;
+   z-index: -1;
+}
+.container{
+   max-width: 1100px;
+   margin: 0 auto;
+   overflow: hidden;
+   height: 100%;
+   display: grid;
+   grid-template-columns: repeat(9, 1fr);
+   grid-template-rows: repeat(9, 1fr);
+}
+/* Header */
+header{
+   grid-column: 1/ span 10;
+   grid-row: 1/ span 1;
+
+}
+header div{
+   display: grid;
+   height: 100%;
+   justify-content: center;
+   align-content: center;
+   text-align: center;
+   font-size: 3.3rem;
+   font-weight: 900;
+   padding-top: 1rem;
+   transform: scaleY(1.5);
+}
+/* Clock */
+#clock{
+   grid-column: 4/ span 3;
+   grid-row: 4/ span 2;
+}
+#clock div{
+   display: grid;
+   height: 100%;
+   justify-content: center;
+   align-content: center;
+   font-size: 5rem;
+   font-weight: 900;
+   transform: scaleY(2.7);
+   white-space: nowrap;
+}
+/* To-Do */
+#to-do{
+   grid-column: 1/ span 2;
+   grid-row: 3/ span 6;
+}
+#to-do div{
+   display: grid;
+   height: 100%;
+   align-content: flex-start;
+   padding-left: 1rem;
+}
+#to-do div h2{
+   font-size: 2rem;
+   font-weight: 900;
+   transform: scaleY(1.5);
+   text-decoration: underline;
+   margin-bottom: 0.25rem;
+}
+#to-do div p{
+   font-size: 1.7rem;
+   font-weight: 900;
+   padding: 0.5rem;
+}
+/* Date */
+#date {
+   grid-column: 8/ span 2;
+   grid-row: 2/ span 1;
+}
+#date div{
+   display: grid;
+   padding-right: 3rem;
+   height: 100%;
+   justify-content: center;
+   align-content: center;
+   font-size: 2rem;
+   font-weight: 900;
+   transform: scaleY(1.3);
+   white-space: nowrap;
+}
+/* Calculator */
+#calculator{
+   grid-column: 7/ span 3;
+   grid-row: 6/ span 3;
+}
+#calculator input{
+   width: 100%;
+   padding: 0.5rem 0;
+   text-align: right;
+   background: rgba(9, 9, 9, 0.2);
+   font-size: 3.2rem;
+}
+button{
+   font-size: 3.4rem;
+}
+.cal-btn{
+   color: #ccc;
+   background: rgba(9, 9, 9, 0.2);
+}
+#calculator .calc-grid{
+   display: grid;
+   grid-template-columns: repeat(4, 1fr);
+}
+
+@media(max-width: 900px){
+   header{
+      grid-column: 2/ span 7;   
+   }
+   #clock{
+      grid-column: 2/ span 3;
+      grid-row: 3/ span 1;
+   }
+   #date {
+      grid-column: 2/ span 2;
+      grid-row: 4/ span 1;
+   }
+   #calculator{
+      grid-column: 7/ span 3;
+      grid-row: 3/ span 3;
+   }
+   #to-do{
+      grid-column: 2/ span 6;
+      grid-row: 6/ span 3;
+   }
+}
+@media(max-width: 750px){
+   .container{
+      grid-template-columns: repeat(6, 1fr);
+      grid-template-rows: repeat(20, 1fr);
+   }
```

Comment:
hm... this doesn't really work too well. to me, you just need 1 column and 4/5 rows. place each child element of container in a row. and maybe make the text smaller. it is on a phone.